### ISSUE TITLE
OneDrive as cloud provider?

### ISSUE THREAD

**Issue by vmcov** [fsantini/KoboCloud#90]
Hi! Any possibility to add OneDrive?

**Comment by SgtDinosaur** [comment]
I share this request!

**Comment by nemo4444** [comment]
Yes! OneDrive is indeed needed. Support will be much appreciated.

**Comment by dsommers** [comment]
This can probably be closed in favour of #145, where there is more on-going discussions.

TL;DR:  Check out the [KoboCloud-rclone](https://github.com/marklar423/KoboCloud-rclone/) fork, it should have [OneDrive support via rclone](https://rclone.org/onedrive/).
